app/models/user.rb

```diff
@@ -0,0 +1,5 @@
+class User < ApplicationRecord
+  validates :username, presence: true
+  validates :email, presence: true
+  validates :password, presence: true
```

Comment:
You could add all on a straight line like => `validates :username, :email, :password, presence: true`